server/boot/customRoutes.js

```diff
@@ -1,8 +1,8 @@
 var loopback = require("loopback");
-module.exports = function(app) {
+module.exports = function (app) {
```

Comment:
I think these changes throughout are because your prettier is overwriting the project prettier settings. I'd make sure there isn't a reason your extension is changing for the backend, or maybe we need to set a prettier config file so its standardized if it isn't already.